src/workloads/execution/MultiPlanning.yml

```diff
@@ -107,7 +107,7 @@ AutoRun:
           - standalone
           - standalone-80-feature-flags
           - standalone-all-feature-flags
-          - standalone-classic-query-engine
+          - query-engine-classic
```

Comment:
Should this be `replica-query-engine-classic`?

Reply:
I thought I caught all of these, but apparently not :')